pulpcore/cli/rpm/distribution.py

```diff
@@ -67,6 +68,13 @@ def distribution(ctx: click.Context, pulp_ctx: PulpCLIContext, distribution_type
             "auto-distribute."
         ),
     ),
+    pulp_option(
+        "--generate-repo-config/--no-generate-repo-config",
+        help=_(
```

Comment:
```suggestion
        "--generate-repo-config/--no-generate-repo-config",
        default=None,
        help=_(
```
I guess, this is needed to make the option a tri-state...